masonry/src/widget/button.rs

```diff
@@ -104,15 +106,15 @@ impl Widget for Button {
             match event.action {
                 accesskit::Action::Default => {
                     ctx.submit_action(Action::ButtonPressed(PointerButton::Primary));
-                    ctx.request_paint();
                 }
                 _ => {}
             }
         }
     }
 
     fn on_status_change(&mut self, ctx: &mut LifeCycleCtx, _event: &StatusChange) {
-        ctx.request_paint();
+        // Changes in hovered/focused status impact appearance, but not accessibility node
+        ctx.request_paint_only();
```

Comment:
Changing the focus status *does* change the accessibility node - https://github.com/linebender/xilem/blob/3ba0df6064a48a92a3efab51455efd378776d33e/masonry/src/passes/accessibility.rs#L119.

However, requesting the accessibility update in that case is not the responsibility of this code, so this is still true.
One thing I'm not sure of - does this update actually happen?

Reply:
Good catch. It would have to happen in `run_update_focus_pass`, and it currently doesn't.